### DISCUSSION TITLE
404 error : There isn't a GitHub Pages site here

### DISCUSSION THREAD

**Discussion by akanksha26281** [community/community#21568] [Other Feature Feedback, Questions, & Ideas]
<p>I created a project and uploaded it on github but now when visiting it I am getting an 404 error here the link to my site :-  <a href="https://akanksha26281.github.io/Tic-tac-toe/" rel="noopener nofollow ugc">https://akanksha26281.github.io/Tic-tac-toe/</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/5/05ba2aa2f48e0e89798a7b01bbcc50700c9fa5cd.png" data-download-href="/uploads/short-url/OFfkMZp27ZiN2BiTNB2wrpXtJr.png?dl=1" title="Screenshot (133)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75932823/181065637-bb3511a6-f20d-4d6a-a47b-e40b106d1cb0.png" alt="Screenshot (133)" data-base62-sha1="OFfkMZp27ZiN2BiTNB2wrpXtJr" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/05ba2aa2f48e0e89798a7b01bbcc50700c9fa5cd_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/5/05ba2aa2f48e0e89798a7b01bbcc50700c9fa5cd_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/5/05ba2aa2f48e0e89798a7b01bbcc50700c9fa5cd_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/05ba2aa2f48e0e89798a7b01bbcc50700c9fa5cd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (133)</span><span class="informations">1920×1080 212 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/akanksha26281">@akanksha26281</a>, welcome to the GitHub Support Community! GitHub Pages looks for an <code>index.html</code>, <code>index.md</code>, or <code>README.md</code> file to serve by default. Your repository doesn’t have any of these files so your Pages site will show a 404.</p>
<p>Your site is online at the following link:</p>
<p><a href="https://akanksha26281.github.io/Tic-tac-toe/1_tic-tac-toe.html">https://akanksha26281.github.io/Tic-tac-toe/1_tic-tac-toe.html</a></p>
<p>To fix this you’ll need to rename your <code>1_tic-tac-toe.html</code> to <code>index.html</code> and then it’ll show up without needing to add the filename to the end of your URL.</p>

**Comment by akanksha26281** [comment]
<p>Thanks <a class="mention" href="/thomasshaped">@thomasshaped</a> for your help my previous issue is solved. Can you tell me why my other two css and js file are not loading :sweat_smile:</p>

**Comment by otter-computer** [comment, marked as answer]
<p>It looks like just a filename issue for both of them! Your code is looking for this file:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="tic-tac-toe.css"&gt;
</code></pre>
<p>But the actual name of that file is:</p>
<pre><code class="lang-auto">2_tic-tac-toe.css
</code></pre>
<p>It’s the same issue with your JS file too. You’ll need to rename bot of these so they match the filenames in your HTML and then they’ll load :tada:</p>

**Comment by akanksha26281** [comment]
<p>Oops thanks, my issue is solved : )</p>

**Comment by Alaoma007** [comment]
<p>Hi I followed the same procedures that you gave akanksha but am still having my page not loading up. it still showing the 404 error… This is the link to my site:-<a href="https://craziefingers.github.io/" rel="noopener nofollow ugc">https://craziefingers.github.io/</a></p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/1/b1c1224ab5c9b77cee2dc989b3a51b5478d5138b.jpeg" data-download-href="/uploads/short-url/pmuiADG74NTuJUHsLe0KOemSVWH.jpeg?dl=1" title="Screen Shot 2021-05-21 at 1.22.36 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181065640-ca286bf2-9482-4ca3-9d02-d6ee04a00877.jpeg" alt="Screen Shot 2021-05-21 at 1.22.36 PM" data-base62-sha1="pmuiADG74NTuJUHsLe0KOemSVWH" width="587" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b1c1224ab5c9b77cee2dc989b3a51b5478d5138b_2_587x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b1c1224ab5c9b77cee2dc989b3a51b5478d5138b_2_880x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b1c1224ab5c9b77cee2dc989b3a51b5478d5138b_2_1174x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/1/b1c1224ab5c9b77cee2dc989b3a51b5478d5138b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-05-21 at 1.22.36 PM</span><span class="informations">2680×2280 1.13 MB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>it’s already working, right? I checked it just now</p>

**Comment by FlowKeyz1996** [comment]
<p>Hi, I tried hosting my website on github pages and my repo has an index.html file and also a readme file but I am still getting back an error after waiting for five hours now.</p>

**Comment by FlowKeyz1996** [comment]
<p>Heres a link <a href="https://github.com/FlowKeyz1996/gym-web" class="inline-onebox" rel="noopener nofollow ugc">GitHub - FlowKeyz1996/gym-web</a></p>

**Comment by Aeka86** [comment]
hey everyone i am having similar issue but my file names are correct and i'm trying to open the expedition file when i deploy it the about part is opening but when i press "expeditions" it shows that error message, it's working on vs code but not in the github for some reason here is a link to my site https://github.com/Aeka86/Magzhan-Website

**Reply by namuwaya** [reply]
please help me with mine its showing 404 error
namuwaya/namuwaya.github.io
